library/aws/shared.ts

```diff
@@ -496,7 +496,7 @@ export function createNetworkResources(
       tfgen,
       "vpc_logs_role",
       {
-        name: "vpc_logs_role",
+        name: `${tfgen.nameContext().join("_") + "_"}vpc_logs_role`,
```

Comment:
move `_` into the right part. Like `${tfgen.nameContext().join("_")}_vpc_logs_role`